src/pipx/commands/inject.py

```diff
@@ -47,7 +47,7 @@ def inject_dep(
     #   zip file, or tar.gz file.
     if package_name is None:
         package_name = package_name_from_spec(
-            package_spec, venv.python, pip_args=pip_args, verbose=verbose
+            package_spec, venv.python_path, pip_args=pip_args, verbose=verbose
```

Comment:
`venv.python_path` seems to be of type `Path`, which breaks the lint because `package_name_from_spec` requires `str`. You should convert `Path` or add the possibility to pass it in the function itself.

Reply:
Thanks for the heads up, @chrysle! Just fixed it 😃